06_gpu_and_ml/text-to-video/mochi.py

```diff
@@ -28,338 +27,147 @@
     .pip_install(
-        "git+https://github.com/genmoai/models.git@075b6e36db58f1242921deff83a1066887b9c9e1"
-    )
+        "git+https://github.com/huggingface/diffusers",
+        "git+https://github.com/huggingface/transformers"
+    ).env({"HF_HUB_ENABLE_HF_TRANSFER": "1"})
 )
 
-app = modal.App("example-mochi")
-
-with image.imports():
-    import numpy as np
-    import ray
-    import torch
-    from einops import rearrange
-    from mochi_preview.handler import MochiWrapper
-    from PIL import Image
-    from tqdm import tqdm
-
-# ## Saving model weights and outputs
-
-# Mochi weighs in at ~80 GB (~20B params, released in full 32bit precision)
-# and can take several minutes to generate videos.
-
-# On Modal, we save large or expensive-to-compute data to
+# ## Saving outputs
+# On Modal, we save large or expensive-to-compute data to 
 # [distributed Volumes](https://modal.com/docs/guide/volumes)
-# so that they are accessible from any Modal Function
-# or downloadable via the Modal dashboard or CLI.
 
-model = modal.Volume.from_name("mochi-model", create_if_missing=True)
-outputs = modal.Volume.from_name("mochi-outputs", create_if_missing=True)
-
-MODEL_CACHE = Path("/models")  # remote path for saving the model
+VOLUME_NAME = "mochi-outputs"
+outputs = modal.Volume.from_name(VOLUME_NAME, create_if_missing=True)
 OUTPUTS_PATH = "/outputs"  # remote path for saving video outputs
 
-# We download the model using the `hf-transfer`
-# library from Hugging Face and additionally download
-# the text encoder (Google's T5 XXL) using `transformers`.
-
-# This can takes five to thirty minutes, depending on traffic
-# and network speed.
-
-# If you want to launch the download first,
-# before running the rest of the code,
-# use the following command from the folder containing this file:
-
-# ```bash
-# modal run --detach mochi::download_model
-# ```
-
-# The `--detach` flag ensures the download will continue
-# even if you close your terminal or shut down your computer
-# while it's running.
-
-download_image = (
-    modal.Image.debian_slim(python_version="3.11")
-    .pip_install(
-        "huggingface_hub",
-        "hf-transfer",
-        "torch",
-        "transformers",
-        "sentencepiece",
-    )
-    .env(
-        {
-            "HF_HUB_ENABLE_HF_TRANSFER": "1",
-            "HF_HOME": str(MODEL_CACHE / "huggingface"),
-        }
-    )
-)
-
-image = image.env(  # so we look for the model in the right place
-    {"HF_HOME": str(MODEL_CACHE / "huggingface")}
-)
-
-
-@app.function(
-    volumes={MODEL_CACHE: model},
-    timeout=2 * HOURS,
-    image=download_image,
-)
-def download_model(
-    model_revision: str = "8e9673c5349979457e515fddd38911df6b4ca07f",
-):
-    from huggingface_hub import snapshot_download
-    from transformers import T5EncoderModel, T5Tokenizer
+MINUTES = 60
+HOURS = 60 * MINUTES
 
-    model.reload()
-    print("🍡 downloading Mochi model")
+# ## Setting up our Mochi class
```

Comment:
Put a space after headers and after paragraphs -- renders the same on the website and also gives a bit of breathing room in the code.

Reply:
I.e. delete the `#` in L62, but leave the `\n`.